etc/rc.local

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh -e
+#
+# rc.local
+#
+# This script is executed at the end of each multiuser runlevel.
+#
+
+nohup sudo -H -u krun python krun.py --resume --reboot /home/krun/krun/examples/example.krun
```

Comment:
I'm not sure, but the script needs to `exit 0` to work with the init framework (I need to test this later today)